common/flatpak-run.c

```diff
@@ -3262,37 +3301,8 @@ add_monitor_path_args (gboolean use_session_helper,
     }
   else
     {
-      /* /etc/localtime and /etc/resolv.conf can not exist (or be symlinks to
-       * non-existing targets), in which case we don't want to attempt to create
-       * bogus symlinks or bind mounts, as that will cause flatpak run to fail.
-       */
-      if (g_file_test ("/etc/localtime", G_FILE_TEST_EXISTS))
-        {
-          char localtime[PATH_MAX + 1];
-          ssize_t symlink_size;
-
-          symlink_size = readlink ("/etc/localtime", localtime, sizeof (localtime) - 1);
-          if (symlink_size > 0)
-            {
-              localtime[symlink_size] = 0;
-              add_args (argv_array,
-                        "--symlink", localtime, "/etc/localtime",
-                        NULL);
-            }
-          else
-            {
-              add_args (argv_array,
-                        "--bind", "/etc/localtime", "/etc/localtime",
-                        NULL);
-            }
-        }
-
-      if (g_file_test ("/etc/resolv.conf", G_FILE_TEST_EXISTS))
-        {
-          add_args (argv_array,
-                    "--bind", "/etc/resolv.conf", "/etc/resolv.conf",
-                    NULL);
-        }
+      add_unmonitored_config_file_args ("/etc/resolv.conf", argv_array);
```

Comment:
Hmm.. no, that's not the case since we'd only make a symlink if the `g_str_has_prefix (target_canonical, "/usr")` condition, used inside `add_unmonitored_config_file_args()`, is true.

And as far as I can see in my other machines, that should not be the case for resolv.conf:
  * In my Fedora 25 and Debian Testing: `/etc/resolv.conf -> /var/run/NetworkManager/resolv.conf`
  * In a server I have with Ubuntu 14.04: `/etc/resolv.conf -> ../run/resolvconf/resolv.conf`
  * In my (debian-based) VPS and another server: `/etc/resolv.conf' is a regular file

Maybe I'm missing something here again?